frontend/lobby/waiting/index.html

```diff
@@ -16,14 +16,15 @@ <h1 class="display-1">
             </h1>
             <div id="waiting-for-opponent" class="row">
                 <div id="game-status" class="col-12">
-                    Waiting for opponent
                 </div>
             </div>
             <template id="joinTemplate">
                 <div id="join-link" class="col-12">
                 </div>
             </template>
         </div>
+        <script src="/config/js-yaml.min.js"></script>
```

Comment:
Can't find this file.

Reply:
it's in `frontend/config/js-yaml.min.js`.

You have to be on this branch to see it.

Reply:
I'd bet it's not a file you handwritted...

Reply:
it's a library 😂